src/components/BulkEnrollmentPage/BulkEnrollButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Button } from '@edx/paragon';
+import { BookOpen } from '@edx/paragon/icons';
+
+/**
+ * Bulk action button, meant to be used in License management page to initiate Bulk Enrollment Dialog
+ */
+const BulkEnrollButton = ({ learners, handleEnrollment }) => (
+  <Button
+    variant="primary"
+    onClick={handleEnrollment}
+    iconBefore={BookOpen}
+    disabled={learners?.length < 1}
+  >
+    Enroll ({learners?.length })
+  </Button>
+);
+
+BulkEnrollButton.propTypes = {
+  handleEnrollment: PropTypes.func.isRequired,
+  learners: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
```

Comment:
nit: I think this can just be `PropTypes.arrayOf(PropTypes.string).isRequired` and we shouldn't need the optional chaining on learners.

Reply:
ah ok the isRequired inside you mean? yes I think we can get rid of that, thanks

Reply:
agreed, don't need that extra isRequired